content/zh/docs/hertz/tutorials/basic-feature/middleware/etag.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "Etag"
+date: 2023-02-11
+weight: 14
+description: >
+---
+
+`etag`（实体标签）HTTP响应头是一个资源的特定版本的标识符。它让缓存更有效，节省带宽，因为如果内容没有改变，网络服务器不需要重新发送完整的响应。此外，`etag` 有助于防止资源的同时更新互相覆盖（"半空碰撞"）。
```

Comment:
- add a description of the hertz etag middleware instead of just about etag
- refer to https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Headers/ETag
- the same goes for English document.